extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -400,7 +436,17 @@ class MembershipsButtonEdit extends Component {
 											'jetpack'
 										) }
 									</p>
-									<Button isDefault isLarge href={ connectURL } target="_blank">
+									<Button
+										isDefault
+										isLarge
+										disabled={ ! stripeConnectUrl }
+										href={ stripeConnectUrl }
+										target={
+											// If we know the postId, we'll try to return to the editor.
+											// Otherwise the flow will continue externally.
+											postId ? undefined : '_blank'
```

Comment:
Consider that on .com we'll be in an iframe, so this should prolly be `_top` instead of `undefined`?